content/guides/understanding-midterm-pricing.md

```diff
@@ -0,0 +1,82 @@
+# Understanding Midterm Pricing
+
+## Assumptions
+
+The length of stay of 31 days is considered by BookingSync as a threshold between short-term and midterm stay.
+Since the midterm stay is no longer the subject of short-term rates rules, the price is calculated differently.
+
+Here is a comparison of what's supported and what's not:
+
+**Price factors**  | **Short-term** | **Midterm**
+-------------------|----------------|------------
+Rates rules (availability, price, occupancy) | Supported | Not supported
+-------------------|----------------|------------
+Seasonality | Supported (via Seasons + Periods or Nightly Rate Map) | Supported (via Midterm Rate Map)
+-------------------|----------------|------------
+Price increase per application | Supported | Supported
+{: class="table table-bordered"}
+
+Of course user can still opt out from midterm pricig (by chnaging Rental's `mid_term_pricing_active` [attribute](/reference/endpoints/rentals/) to `false`)
+
+## What happens when midterm pricing is disabled?
+
+Even though consulted with our users, we acknowledge the fact that for some reason this approach may not work for all of them, therefore they can opt-out of midterm pricing.
+
+If this is the case, there are still ways to allow midterm bookings:
+
+  * Live quote (via [rentals_search endpoint](/reference/endpoints/rentals/#search-rentals)) - includes all rates rules, yet cannot be cached and may affect integration performance and overall user experience
+  * [Rates](/reference/endpoints/rates/) - it's a nightly price, independent of the length of stay. To protect users from losing money, it calculates the highest possible daily pricing. Because of that midterm pricing discounts (such as a discount for a stay over 30 days will not be applied here) will not be included here, which may lead to less attractive pricing
```

Comment:
`To protect users from losing money, it calculates the highest possible daily pricing.` - are you sure about this? some rates rules are not applied on rates (because we don't have a reference to the current date), so the price might actually be lower

Reply:
@StoneFrog can you help here ?

Reply:
well, if there are LOS modifiers - it raises minimum stay to ensure correct pricing, so I was referring mostly to that. Also as far as I remember if there was a price increase - it was applied on rate, whereas price decrease wasn't.

Reply:
so I'd say it's valid in the context of LOS rates rules, excluding stay_at_least :trollface:

Reply:
So in the end, this is not fully true, no? :D I would rather link here to http://developers.bookingsync.com/guides/pricing-concepts#rates-and-rates-rules-not-recommended

Reply:
And did you read it?
```
Rates are calculated based on the seasons/periods/nightly rates maps/rates rules, so those resources should not be used for any calculation, they should be considered as an implementation detail.
```
It actually suggests to use rates, and what I'm trying to explain here, is that they have their own limitations and why enabling Midterm pricing is a better option.

I don't think there are any resources that explain in detail how rates are calculated what is and what is not included.

Reply:
Oh it's a different paragraph 🤦 OK for linking then :D